src/overlays/actors/ovl_Bg_Spot18_Basket/z_bg_spot18_basket.c

```diff
@@ -21,39 +39,396 @@ const ActorInit Bg_Spot18_Basket_InitVars = {
     (ActorFunc)BgSpot18Basket_Update,
     (ActorFunc)BgSpot18Basket_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot18_Basket/func_808B7710.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot18_Basket/func_808B7770.s")
+static ColliderJntSphItemInit sJntSphItemsInit[2] = {
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+        { 0, { { 0, 2040, 0 }, 54 }, 100 },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { 1, { { 0, 1400, 0 }, 13 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x09, 0x20, COLSHAPE_JNTSPH },
+    2,
+    sJntSphItemsInit,
+};
+
+static s16 D_808B85C8[] = { 32768, 10922, 54613, 0 };
+
+void func_808B7710(Actor* thisx, GlobalContext* globalCtx) {
+    BgSpot18Basket* this = THIS;
+
+    Collider_InitJntSph(globalCtx, &this->colliderJntSph);
+    Collider_SetJntSph(globalCtx, &this->colliderJntSph, &this->dyna.actor, &sJntSphInit, &this->colliderJntSphItem);
+    this->dyna.actor.colChkInfo.mass = 255;
```

Comment:
```suggestion
    this->dyna.actor.colChkInfo.mass = 0xFF;
```
for the two special mass values 0xFE and 0xFF we use hex, other values should indeed be decimal iirc